### ISSUE TITLE
bugs when training DAIR-V2X

### ISSUE THREAD

**Issue by cuge1995** [yifanlu0227/CoAlign#11]
frame_info['cooperative_label_path']
KeyError: 'cooperative_label_path'

**Comment by yifanlu0227** [comment]
Could you please check the file `v2x_c/cooperative-vehicle-infrastructure/cooperative/data_info.json` in your downloaded dataset? It should exist.
![截屏2023-08-17 18 54 17](https://github.com/yifanlu0227/CoAlign/assets/45688237/3ceb4693-80b5-4e69-aaa2-fa91a9eba5e8)

**Comment by cuge1995** [comment]
Yes, it exists. But the training ends during training:
```
[epoch 0][306/601] || Loss: 2.0229 || Conf Loss: 0.8440 || Loc Loss: 1.0396 || Dir Loss: 0.1394 || IoU Loss: 0.0000
[epoch 0][307/601] || Loss: 2.5544 || Conf Loss: 0.8616 || Loc Loss: 1.5484 || Dir Loss: 0.1444 || IoU Loss: 0.0000
Traceback (most recent call last):
  File "opencood/tools/train.py", line 198, in <module>
    main()
  File "opencood/tools/train.py", line 106, in main
    for i, batch_data in enumerate(train_loader):
  File "/home/selfdriving/anaconda3/envs/opencood/lib/python3.7/site-packages/torch/utils/data/dataloader.py", line 517, in __next__
```

**Comment by yifanlu0227** [comment]
Did you follow the **step3** in [DAIR_V2X_C download](https://siheng-chen.github.io/dataset/dair-v2x-c-complemented/)?

**Comment by cuge1995** [comment]
Yes

**Comment by yifanlu0227** [comment]
Well, that's weird. If you can provide the missing frame id, it will be of great help for us to solve the problem.

**Comment by HuangZhe885** [comment]
I also encountered a similar problem, and the error was 
<img width="1438" alt="image" src="https://github.com/yifanlu0227/CoAlign/assets/44192081/059e797b-bd4d-4e8b-a80f-5f1d15e58aa8">


So I deleted the ’_backup‘ in ’label_world_backup‘ and it ran successfully, but I don’t know if this is correct.

        data[0]['params']['vehicles_front'] = read_json(os.path.join(self.root_dir,frame_info['cooperative_label_path'].replace("label_world", "label_world_backup")))

**Comment by yifanlu0227** [comment]
@HuangZhe885 Solved in #20 . The dataset website has been updated.
